lib/utilities/find-build-file.js

```diff
@@ -2,15 +2,23 @@
 const findUp = require('find-up');
 const path = require('path');
 
-module.exports = function(file, dir) {
-  let buildFilePath = findUp.sync(file, { cwd: dir });
+module.exports = function(file, args = {}) {
+  let { isEmberCLIAddon, projectRoot } = args;
+  let options = { cwd: projectRoot };
 
-  // Note: In the future this should throw
+  if (isEmberCLIAddon) {
+    // for addons, the ember-cli-build.js file will be at tests/dummy.
+    // find-up will only find files upwards in the directory.
+    // To solve this, resolve the path for ember-cli for Ember Addons.
+    options.cwd = path.resolve(process.cwd(), `${projectRoot}/tests/dummy`);
```

Comment:
I don't like this hard-coding, it makes (for example) having multiple dummy apps impossible...